jetty-core/jetty-deploy/src/main/config/modules/deployer-standard.mod

```diff
@@ -1,5 +1,6 @@
 [description]
-This module enables the direct Deployer, which simply adds and starts a context.
+This module enables the standard Deployer, which adds and starts ContextHandler
```

Comment:
Huh? What is the `deployer-standard` module? How is this different to the `deployment-scanner` module? Shouldn't we have just one `deployer` module, and if users want something different they come up with their own module that configures their own Scanner etc.